bedrock/firefox/templates/firefox/home.html

```diff
@@ -190,7 +70,7 @@ <h2 class="section-title">{{ _('Seriously Private Browsing') }}</h2>
 
       <ul>
         <li class="private-browsing">
-          <h3>{{ _('Private Browsing') }}</h3>
+          <h3><a href="{{ url('firefox.features.private-browsing') }}">{{ _('Private Browsing') }}</a></h3>
```

Comment:
It would be nice if these links had some visual identifier / hover state?

Reply:
That's out of scope for this PR.

Reply:
Hmm, ok  - but let's file an issue to follow up? Whilst adding links for better internal linking is worthwhile, we shouldn't be adding them only for robots and leaving them inaccessible to users.